src/picongpu/include/particles/ionization/None/AlgorithmNone.hpp

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright 2014 Marco Garten
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "types.h"
+
+/* IONIZATION ALGORITHM */
+
+namespace picongpu
+{
+    namespace particles
```

Comment:
our weird [namespace white space rules](https://github.com/ComputationalRadiationPhysics/picongpu/wiki/Coding-Guide-Lines#1-spaces-and-indentation) are sexy